common/pkg/testutils/capitest/request/items.go

```diff
@@ -45,15 +45,24 @@ func NewRequestItem(
 	}
 }
 
-func NewKubeadmConfigTemplateRequestItem(uid types.UID) runtimehooksv1.GeneratePatchesRequestItem {
+func NewKubeadmConfigTemplateRequestItem(
+	uid types.UID,
+) runtimehooksv1.GeneratePatchesRequestItem {
+	return NewKubeadmConfigTemplateRequest(uid, kubeadmConfigTemplateRequestObjectName)
+}
+
+func NewKubeadmConfigTemplateRequest(
```

Comment:
How about something like this? I don't think dropping the "Item" is clear what the difference is
```suggestion
func NewNamedKubeadmConfigTemplateRequestItem(
```